designs/2019-worker-threads/README.md

```diff
@@ -22,27 +22,28 @@ This RFC has two steps.
 
 - Adds `--concurrency` CLI option to specify the number of worker threads. Defaults to `1`.
 - Adds `concurrency` constructor option to `ESLint` class to specify the number of worker threads. Defaults to `1`.
+- Adds `disallowWorkerThreads` option to plugins. Defaults to `false`.
 - Changes the behavior of `ESLint#executeOnFiles(patterns)` method.
 
 **The step 2 (semver-major):**
 
-- Changes the default of the `--concurrency` CLI option to `"auto"`.
+- Changes the default of the `--concurrency` CLI option to `auto`.
 - Changes the default of the `concurrency` constructor option to `"auto"`.
 
-### [STEP 1] New `--concurrency` CLI Option
+### [STEP 1] New `--concurrency` CLI option
 
-The `--concurrency` has an argument. The argument should be a positive integer or `"auto"`. Defaults to `1`.
+The `--concurrency` has an argument. The argument should be a positive integer or `auto`. Defaults to `1`.
 
 - If `1` is present, ESLint does linting in the main thread.
 - If another integer is present, ESLint creates this number of worker threads then the worker threads do linting.
 
-If `auto` is present, ESLint estimates the best value from the number of target files. It's the number of target files divided by the constant `128`, but `os.cpus().length` at most.
+If `auto` is present, ESLint estimates the best value from the number of target files. It's the number of target files divided by the constant [128](#constants), but `os.cpus().length` at most.
```

Comment:
I'm a bit confused by this sentence. Do you mean we will use `os.cpus().length` to determine number of workerThreads, but no more then 128 at any given time? I.e. if somebody is linting on a machine with 256 cpus, we will only use 128 workerThreads?

Reply:
No. You have missed "the number of target files." This sentence means [the below expression](https://github.com/eslint/rfcs/pull/42/files#diff-974e1ce39d3337e6a0bc28fc25fb8d11R42-R44). If this sentence is wrong in English grammar, would you correct it?

Reply:
I’d say maybe:
> It’s `Math.max(number of target files / [128](#constants), os.cpus().length)`

Reply:
Sure, such an expression exists the below of this sentence :)